src/main/java/net/projectzombie/region_rotation/controller/BaseStateCommands.java

```diff
@@ -0,0 +1,144 @@
+                    String backupRegionName = args[3];
+                    World backupWorld = Bukkit.getWorld(args[4]);
+
+                    if (world != null && backupWorld != null
+                        && WGBukkit.getRegionManager(world).getRegion(regionName) != null
+                        && WGBukkit.getRegionManager(backupWorld)
+                                    .getRegion(backupRegionName) != null)
+                    {
+                        baseState = new BaseState(regionName,
+                                                  world.getUID(),
+                                                  backupRegionName,
+                                                  backupWorld.getUID());
+                        success = StateController.instance().addBaseState(baseState);
+                    }
+                }
+                sender.sendMessage(addBaseState(baseState, success));
+            }
+            else if (args[0].equals(ADD_ALT_BASESTATE_CMD) &&
+                    sender.hasPermission(ADD_ALT_BASESTATE_PERM))
+            {
+                boolean success = false;
+                if (args.length == 4)
+                {
+                    String regionName = args[1];
+                    String altRegionName = args[2];
+                    World altRegionWorld = Bukkit.getWorld(args[3]);
+                    if (altRegionWorld != null)
+                    {
+                        success = StateController.instance().addAltState(regionName,
+                                altRegionName,
+                                altRegionWorld.getUID());
+                    }
+                }
+                sender.sendMessage(addAltState(success));
+            }
+            else if (args[0].equals(REMOVE_BASESTATE_CMD) &&
+                     sender.hasPermission(REMOVE_BASESTATE_PERM))
+            {
+                boolean success = false;
+                if (args.length == 2)
+                {
+                    String regionName = args[1];
+                    success = StateController.instance().removeBaseStateFully(regionName);
+                }
+                sender.sendMessage(removeBaseState(success));
+            }
+            else if (args[0].equals(RESET_BASESTATE_CMD) &&
+                    sender.hasPermission(RESET_BASESTATE_PERM))
+            {
+                boolean success = false;
+                if (args.length == 3)
+                {
+                    String regionName = args[1];
+                    boolean broadcast = Boolean.valueOf(args[2]);
+                    success = StateController.instance().resetBaseState(regionName, broadcast);
+                }
+                sender.sendMessage(resetBaseState(success));
+            }
+            else if (args[0].equals(ROTATE_BASESTATE_CMD) &&
+                    sender.hasPermission(ROTATE_BASESTATE_PERM))
+            {
+                boolean success = false;
+                if (args.length == 5)
```

Comment:
It displays the rotate help, if success is false. And if args is < 5 then it should do the help one. The commandHelp() displays the <add:addalt:remove:reset:rotate:list>, is that what you want it to display?